include/ocpp/v201/charge_point_callbacks.hpp

```diff
@@ -151,5 +151,14 @@ struct Callbacks {
     std::optional<std::function<void(const RunningCost& running_cost, const uint32_t number_of_decimals,
                                      std::optional<std::string> currency_code)>>
         set_running_cost_callback;
+
+    /// \brief Callback function is called when a reservation request is received from the CSMS
+    std::optional<std::function<ReserveNowStatusEnum(
```

Comment:
Any reason why you don't just forward the request like we do with the other callbacks? Saves a bunch of arguments ;)

Reply:
Hmm I kept it quite the same as in OCPP 1.6. I don't know why the choice there was to do it this way. Better change it then?

Reply:
I would prefer to keep the interface for the 2.0.1 implementation consistent over certain functionality shared between 1.6 and 2.0.1 so yeah I would change it. Just my 2 cents ofcourse.

Reply:
Well it is consistent with 1.6. 

@Pietfried  do you have a preference?

Reply:
Talked to Piet, I will change it to the Reservation type (give me some time :)).

Reply:
done!